tensorrt_llm/_torch/pyexecutor/sampler.py

```diff
@@ -88,6 +88,34 @@
 T = TypeVar("T")
 
 
+@dataclass(kw_only=True)
+class LogProbsState:
+    sampled_vals: torch.Tensor
+    sampled_indices: torch.Tensor
+    sampled_rank: torch.Tensor
+    topk_vals: torch.Tensor
+    topk_indices: torch.Tensor
+
+
+@dataclass(kw_only=True)
+class LogProbsStateList:
+    sampled_vals: list[list[list[float]]]
```

Comment:
Many duplicates of this nested `list[list[list` type, shall we consider using https://docs.python.org/3/library/typing.html#type-aliases ?